new-site/css/style.css

```diff
@@ -245,3 +245,242 @@ textarea {
+  display: block;
+  color: rgb(156, 83, 83);
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.nav-bar a {
+  text-decoration: none;
+  color: #333;
+  font-weight: bold;
+  font-size: 16px;
+}
+.nav-bar li {
+  float: right;
+  display: inline;
+  color: black;
+  margin: 0 15px;
+  border-radius: 10%;
+}
+
+.Navbar a:hover {
+  cursor: pointer;
+  color: rgb(235, 124, 143);
+}
+
+.header .nav-bar a {
+  margin: 0 1rem;
+  font-size: 1.6 rem;
+  color: white;
+}
+
+.header .nav-bar a:hover {
+  color: var(--primary-color);
+  border-bottom: 0.1rem solid var(--primary-color);
+}
+li a,
+.dropbtn {
+  display: inline-block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+li a:hover,
+.dropdown:hover .dropbtn {
+  background-color: rgb(50, 112, 170);
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: rgb(50, 112, 170);
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+  z-index: 1;
+}
+
+.dropdown-content a {
+  color: rgb(95, 131, 197);
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+  text-align: left;
+}
+
+.dropdown-content a:hover {
+  background-color: #f1f1f1;
+}
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+.cards {
+  justify-content: center;
+  margin: 20px auto;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+/* class for all cards */
+.container {
+  width: 360px;
+  height: 170px;
+  border: 5px solid #000000;
+  border-radius: 5px;
+  float: left;
+  margin: 20px;
+}
+
+
+#Angular {
+  width: 360px;
+  padding: 5px 15px;
+  height: 500px;
+  margin-top: 5%;
+  border-radius: 5px;
+  margin-bottom: 5%;
+}
+#Vue {
+  width: 360px;
+  padding: 5px 15px;
+  height: 500px;
+  border-radius: 5px;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+div#React {
+  width: 360px;
+  padding: 5px 15px;
+  border-radius: 5px;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  height: 500px;
+}
+
+#React-Hooks {
+  width: 360px;
+  padding: 5px 15px;
+  font-size: 7px;
+  border-radius: 5px;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  height: 500px;
+
+
+}
+
+/* table */
+table {
+  overflow: auto;
+  width: 100%;
+  border-collapse: collapse;
+  margin-bottom: 10%;
+  font-size: 12px;
+}
+
+th,
+td {
+  font-size: 12px;
+  overflow: auto;
+  border: 1px solid black;
+  padding: 8px;
+  text-align: left;
+}
+
+/* button */
+.button {
+  background-color: rgb(118, 118, 218);
+  margin-left: 70%;
+  margin-bottom: 5%;
+  display: inline-block;
+  border-radius: 5px;
+  padding: 5px;
+  text-decoration: none;
+}
+
+img {
+  width: 100%;
+  height: 200px;
+  background-size: cover;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+}
+
+
+h2 {
+  text-align: center;
+  color: rgb(123, 12, 12);
+  font-family: sans-serif;
+}
+
+
+p {
+  color: dark black;
+  font-size: 15px;
+  margin-bottom: 10px;
+}
+
+/* footer */
+footer {
+  clear: both;
```

Comment:
Clear is for float layouts, so you need to move away from this and use flex or grid.